lib/assets/seeders/seed-table.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
```

Comment:
This seems like a copy past mistake, I couldn't find any reference to this file => deleting it